app/models/miq_server.rb

```diff
@@ -590,11 +590,23 @@ def self.zone_is_modifiable?
     Zone.visible.in_my_region.count > 1
   end
 
+  def self.managed_providers
+    providers = {}
+    ExtManagementSystem.all.each { |ems| providers[ems.type] = ems.class.count }
+
+    providers
+  end
+
   def self.managed_resources
     {
       :vms                     => Vm.active.count,
       :hosts                   => Host.active.count,
       :aggregate_physical_cpus => Host.active.in_my_region.sum(:aggregate_physical_cpus),
+      :providers               => managed_providers,
+      :deployment              => MiqEnvironment::Command.is_appliance? ? "appliance" : "podified",
```

Comment:
I think prefer containerized here as it will also cover docker monolithic deployments (which aren't "pods", per se).  Also I don't really like the phrase "podified" - the rest of the world calls it "containerized" or (also gross) "cloud native".

```suggestion
      :deployment              => MiqEnvironment::Command.is_appliance? ? "appliance" : "containerized",
```